core-bundle/src/Command/DebugFragmentsCommand.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Command;
+
+use Contao\CoreBundle\Fragment\FragmentConfig;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class DebugFragmentsCommand extends Command
+{
+    protected static $defaultName = 'debug:fragments';
+
+    /**
+     * @var array
+     */
+    private $identifiers = [];
+
+    /**
+     * @var FragmentConfig[]
+     */
+    private $configs = [];
+
+    /**
+     * @var array
+     */
+    private $attributes = [];
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
+
+    public function add(string $identifier, FragmentConfig $config, array $attributes): void
+    {
+        $this->identifiers[] = $identifier;
+        $this->configs[$identifier] = $config;
+        $this->attributes[$identifier] = $attributes;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Displays the fragment controller configuration.')
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->io = new SymfonyStyle($input, $output);
+
+        $rows = [];
+        $identifiers = $this->identifiers;
+        natsort($identifiers);
+
+        foreach ($identifiers as $identifier) {
+            $rows[] = [
+                $identifier,
+                $this->configs[$identifier]->getController(),
+                $this->configs[$identifier]->getRenderer(),
+                $this->generateArray($this->configs[$identifier]->getOptions()),
+                $this->generateArray($this->attributes[$identifier]),
+            ];
+        }
+
+        $this->io->title('Contao Fragments');
+        $this->io->table(['Identifier', 'Controller', 'Renderer', 'Render Options', 'Fragment Options'], $rows);
+
+        return 0;
+    }
+
+    private function generateArray(array $values): string
+    {
+        $length = array_reduce(
+            array_keys($values),
+            static function ($carry, $item) {
```

Comment:
I think we should add `: int` as return type here, shouldn't we?

Reply:
added in a0aa306a8b4a74185e375d088116dd57238bd46a